packages/wdio-sauce-service/README.md

```diff
@@ -58,6 +58,33 @@ export.config = {
 };
 ```
 
+If you want to use an existing Sauce Connect tunnel you only need to provide the `tunnelIdentifier` into you capabilites like this
+
+```js
+export.config = {
+    // ...
+    {
+        browserName: 'chrome',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        // Sauce options can be found here https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options
+        'sauce:options': {
+            tunnelIdentifier: 'your-tunnel-name',
+            
+            // If a parent tunnel is needed use this combination
+            tunnelIdentifier: 'ParentTunnelName',
+            parentTunnel: '<username of parent>,
```

Comment:
could this be a bit confusing if `tunnelIdentifier` is set twice? How about we use code tabs here to show an example with and without parent tunnel?

Reply:
Good idea, let me do that